app/boards/shields/settings_reset/settings_reset.overlay

```diff
@@ -8,10 +8,10 @@
 
 / {
     chosen {
-        zmk,kscan = &kscan0;
+        zmk,kscan = &dummy_kscan;
     };
 
-    kscan0: kscan {
+    dummy_kscan: dummy_kscan {
```

Comment:
If we called this settings_reset_kscan, then there would be even less of a chance of it conflicting with something else.